### DISCUSSION TITLE
[PROPOSAL] 🔪 Butchering Proposal

### DISCUSSION THREAD

**Discussion by esp123** [sunflower-land/sunflower-land#2070] [Ideas]
**Butchering Proposal** 

**Fancy some meats?** Presenting... Butchering, where you can ethically butcher animals for raw protein... and more 👀.

**Goals:**
1. Serves as a way to burn resources and SFL.
2. Open more possible food ideas to the game.
3. Open ideas regarding different animal parts being available ingame.
4. I just want to slaughter some chickens. 😈 

**Proposed Mechanics:**
1. Placed Animals:
- Buy *Butchering Knife* in the *Workbench*.
- Equip and tap on the placed animals in map (*only works if hungry*).
- Receive produce.
(*Will not use Butcher Stand*).

2. Butcher Stand:
- Buy *Butchering Knife* in the *Workbench*.
- Buy chickens or Shovel(remove) placed Animals.
- Go to *Butcher Stand*.
- Receive produce.

3. Use both

**Tools**
Butchering Knife
![menu-knife](https://user-images.githubusercontent.com/74337946/213694027-c0e3fa19-b2af-4728-b3c0-5c3de93812f0.png)
![knife1](https://user-images.githubusercontent.com/74337946/213687587-322e5fc8-920c-4580-9029-2c3f7a1e10a2.png)
- Use to butcher animals.
- Can be crafted in *Workbench*.

Requirements: 
1. 0.0625 sfl
2. 3 woods
3. 5 stone

Butchering knife mechanics:
1. *Butchering knife* will have 3-5 uses before it breaks.
4. *Butchering Knife* can be use 4 times(consumes 0.25 in durability) before it breaks.

**Building**
Butcher Stand
![butcher1](https://user-images.githubusercontent.com/74337946/213691032-df28e437-e000-443e-bfb8-4b23ea4c1c31.png)
*Slice, slice*
-Use to butcher animals. (*Proposed mechanic 2 and 3*)

Requirements:
1. Bumpkin level: 20
2. 50 woods
3. 10 stones
4. 5 chickens

**Product**
1. Raw Chicken
![menu-chix](https://user-images.githubusercontent.com/74337946/213697217-5595e1b0-02cd-40f0-b802-41027c815e53.png)
![penk](https://user-images.githubusercontent.com/74337946/213697167-14a17af0-bed5-49e8-bdf4-2f51c44a19fd.png)
![menu-chix1](https://user-images.githubusercontent.com/74337946/213697490-911d6dc8-02ac-49a7-8294-4d7c86566b17.png)
![xhix1](https://user-images.githubusercontent.com/74337946/213697475-c710b073-230f-4058-9725-44ad31bf6fdd.png)
- Lean meat

Requirements:
- 1 chicken
- 1 butchering knife

2. Raw Pork
![menu-pork](https://user-images.githubusercontent.com/74337946/213697760-98d078ef-da5c-46ab-84c5-21b720a38065.png)
![pork](https://user-images.githubusercontent.com/74337946/213697746-d61440a4-3713-4252-8531-48fc88f22e87.png)
- *Lechon*

Requirements:
- 1 pig
- 1 butchering knife

3. Raw Beef
![menu-beef](https://user-images.githubusercontent.com/74337946/213698154-bb3c1852-acb1-47c7-8630-3e7e64d20b56.png)
![beef](https://user-images.githubusercontent.com/74337946/213698108-a7552241-813a-4c10-8382-12136792a121.png)
![menu-beef1](https://user-images.githubusercontent.com/74337946/213698170-cc857298-037f-47f1-996a-993ec334977a.png)
![beef1](https://user-images.githubusercontent.com/74337946/213698125-3df5394d-9369-4836-8934-4a4aea96cd97.png)
- Quite a luxury.

Requirements:
- 1 cow
- 1 butchering knife

Other animal part such as feathers, leathers and horns are for the future proposals.

Feel free to give feedbacks, opinions and criticism. Will mean a lot to me as this is my first time creating one😅.
Special thanks to JC Ei and LittleEins❤️ 
Discord ID: deeefault#0033

**Comment by mmaxxpeinn** [comment]
loving the idea of butchering chickens 

*especially those that don't give mutant chicken **even once*** 😈
